### PR TITLE
fix: Wrong state for Abt Entur SDK when initializing timer

### PR DESCRIPTION
**Pull request by jorelosorio**
Related to: https://github.com/AtB-AS/kundevendt/issues/18509

TimeContextProvider requires AbtClient to be initialized, this fix import some configurations as well initializes the AbtClient.

### CODE REVIEW

File: src/mobile-token/abtClientLogger.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const localLogger: ClientConfig['localLogger'] = {
   },
   warn: (msg, err, metadata?) => {
     const onError = toOnErrorCallback('warning', msg, metadata);
-    Bugsnag.notify(err, onError);
+    if (err) Bugsnag.notify(err, onError);
```
Comment (jorelosorio) [inline src/mobile-token/abtClientLogger.ts:13]: A fix for https://app.bugsnag.com/atb-as/mittatb/errors/66600166df36580008bb5349?filters[user.id]=57f8e086-e0c7-49ea-b6d9-974838db6cb0&event_id=667bcf1f00efc9de65c30000

It uses the same check as in `error`.

Comment (gorandalum) [reply src/mobile-token/abtClientLogger.ts:13]: So we no longer want to notify every warn event to Bugsnag?

**Review by gorandalum — APPROVED** [review]

File: src/mobile-token/abtClientLogger.ts (hunk 1 above)
Comment (tormoseng) [reply src/mobile-token/abtClientLogger.ts:13]: What kind of warnings is this? Do you have any examples?

Comment (jorelosorio) [reply src/mobile-token/abtClientLogger.ts:13]: If error is undefined, we got a wrong error with no warning info.

Comment (tormoseng) [reply src/mobile-token/abtClientLogger.ts:13]: Aha. That one

**Pull request merged**